.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+
+<!-- Thanks for submitting this pull request for KeY -->
+<!-- Since the project has a strict review policy, please make the -->
+<!-- reviewer's job easier by providing the necessary information -->
+<!-- in the text below. The comments may remain since they will be -->
+<!-- invisible when showing the PR. -->
+
+## Related Issue
+
+<!-- Please remove if this PR is not related to an issue. -->
+<!-- Please add number if it is in answer to an issue. -->
+This pull request answers to issue #.
+
+## Intended Change
+<!-- Please give a brief description of what behaviour changes and 
+     why it should be changed. -->
+
+## Reason for the pull request
+
+<!-- one item like: 
+     bugfix, new feature, refactoring, repaired workflow, ... -->
+
+## Touched elements
```

Comment:
I see this as redundant information. The same is tracked by Github under "Files changed".

Reply:
Not quite as mentioned then. But we can drop that.